src/main/webapp/style.css

```diff
@@ -0,0 +1,56 @@
+:root {
+  --failed: #DC4F4F;
```

Comment:
please use lowercase for the constant: https://google.github.io/styleguide/htmlcssguide.html#Capitalization

Reply:
Addressed here d029e00